integration-tests/src/test/scala/tech/beshu/ror/integration/suites/DataStreamApiSuite.scala

```diff
@@ -202,39 +178,96 @@ trait DataStreamApiSuite extends AnyFreeSpec
           }
         }
       }
+      "with data_streams rule should" ignore { // todo enable when search request will be aware of data streams in ACL
```

Comment:
I ignored these tests for now because improvements to the search API are not ready yet

Reply:
because we are preparing to release and searching API is pretty important (basic usage) we cannot merge it. Let's wait for the fix.

Reply:
ref: RORDEV-820